src/resources/views/pages/auth/register.blade.php

```diff
@@ -0,0 +1,154 @@
+        @error('lot_block')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Street -->
+        <label for="street">Street:</label>
+        <input type="text" id="street" name="street">
+        @error('street')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- City -->
+        <label for="city">City:</label>
+        <input type="text" id="city" name="city">
+        @error('city')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Province -->
+        <label for="province">Province:</label>
+        <input type="text" id="province" name="province">
+        @error('province')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Country -->
+        <label for="country">Country:</label>
+        <input type="text" id="country" name="country">
+        @error('country')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Zip Code -->
+        <label for="zip_code">Zip Code:</label>
+        <input type="text" id="zip_code" name="zip_code">
+        @error('zip_code')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Email -->
+        <label for="email">Email:</label>
+        <input type="email" id="email" name="email">
+        @error('email')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Username -->
+        <label for="username">Username:</label>
+        <input type="text" id="username" name="username">
+        @error('username')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Phone Number -->
+        <label for="phone_number">Phone Number:</label>
+        <input type="text" id="phone_number" name="phone_number">
+        @error('phone_number')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Password -->
+        <label for="password">Password:</label>
+        <input type="password" id="password" name="password">
+        @error('password')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Confirm Password -->
+        <label for="password_confirmation">Confirm Password:</label>
+        <input type="password" id="password_confirmation" name="password_confirmation" required>
+        @error('password_confirmation')
+        <div style="color:red;">{{$message}}</div>
+        @enderror
+        <br>
+
+        <!-- Cancel Button -->
+        <a href="/register-page">Cancel</a>
+
+        <!-- Submit Button -->
+        <button>Submit</button>
+
+        <p>Already have an Account? <a href="/" class="underline">Sign In!</a></p>
+
+    </form>
+    @endauth
+</body>
+
+</html>
```

Comment:
trail spacing!

If you see something like this in your PR just add breakline

Reply:
![image](https://github.com/xReddskie/microblog-ojt2/assets/115203179/e4c0678a-0e0d-43da-b045-0d86f7f8709c)